apps/web/src/pages/integrations/constants.ts

```diff
@@ -1,6 +1,16 @@
+import { ChannelTypeEnum } from '@novu/shared';
+
 export enum IntegrationsStoreModalAnalytics {
   CREATE_INTEGRATION_FORM_SUBMIT = 'Create Integration Form Submit - [Integrations Store Modal]',
   UPDATE_INTEGRATION_FORM_SUBMIT = 'Update Integration Form Submit - [Integrations Store Modal]',
   SELECT_PROVIDER_CLICK = 'Select Provider Click - [Integrations Store Modal]',
   CLOSE_MODAL = 'Close Modal - [Integrations Store Modal]',
 }
+
+export const CHANNEL_TYPE_TO_ICON = {
```

Comment:
```suggestion
export const CHANNEL_TYPE_TO_ICON_NAME = {
```

Maybe something like this? because we do not map to the actual icon here, or should it be ok because it is in the `fortawesome` context? (if this is the case maybe we can change the namespace by creating a dedicated file for it?)